doorhole.py

```diff
@@ -377,7 +377,7 @@ def deleteRow(self, qidx):
 			qm.setText(str("This will delete the requirement from disk.\nYou will not be able to recover it unless it's versioned.\n\nAre you absolutely sure?"))
 			qm.setStandardButtons(QMessageBox.Yes | QMessageBox.No)
 			ret = qm.exec()
-			if ret == qm.Yes:
+			if ret == QMessageBox.StandardButtons.Yes.value:
```

Comment:
I investigated the issue a bit (SO + QT docs) and did a quick test: both `QMessageBox.StandardButtons.Yes.value` and  `QMessageBox.Yes` work, while `qm.Yes` does not work anymore.

I would use `QMessageBox.Yes` because it is used also inside `qm.setStandardButtons(QMessageBox.Yes | QMessageBox.No)` and it doesn't trigger a brain interrupt ("wait, what?")